src/ui/templates/insurance/your-business/company-details.njk

```diff
@@ -98,109 +98,154 @@
       </div>
     {% endif %}
 
-    <div class="govuk-grid-column-three-quarters-from-desktop">
-      {{ govukRadios({
-        classes: "govuk-radios--inline",
-        name: "trading-name",
-        fieldset: {
-          legend: {
-            text: CONTENT_STRINGS.TRADING_NAME,
-            isPageHeading: false,
-            classes: "govuk-fieldset__legend--m"
-          },
-          attributes: {
-            'data-cy': 'trading-name'
-          }
-        },
-        items: [
-          {
-            value: "yes",
-            text: "Yes",
+    <form action="{{COMPANY_DETAILS_POST_ROUTE}}" method="POST" novalidate>
+      <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+      <div class="govuk-grid-column-three-quarters-from-desktop">
+        {{ govukRadios({
+          classes: "govuk-radios--inline",
+          idPrefix: FIELDS.YOUR_COMPANY.TRADING_NAME,
+          name: FIELDS.YOUR_COMPANY.TRADING_NAME,
+          fieldset: {
+            legend: {
+              text: CONTENT_STRINGS.TRADING_NAME,
+              isPageHeading: false,
+              classes: "govuk-fieldset__legend--m"
+            },
             attributes: {
-              'data-cy': "trading-name-yes"
+              'data-cy': 'trading-name'
             }
           },
-          {
-            value: "no",
-            text: "No",
+          items: [
+            {
+              value: "yes",
+              text: "Yes",
+              checked: submittedValues[FIELDS.YOUR_COMPANY.TRADING_NAME] === 'yes',
+              attributes: {
+                'data-cy': "trading-name-yes",
+                'aria-label': CONTENT_STRINGS.TRADING_NAME + ' yes radio'
+              }
+            },
+            {
+              value: "no",
+              text: "No",
+              checked: submittedValues[FIELDS.YOUR_COMPANY.TRADING_NAME] === 'no',
+              attributes: {
+                'data-cy': "trading-name-no",
+                'aria-label': CONTENT_STRINGS.TRADING_NAME + ' no radio'
+              }
+            }
+          ],
+          errorMessage: validationErrors.errorList.tradingName and {
+            text: validationErrors.errorList.tradingName.text,
             attributes: {
-              'data-cy': "trading-name-no"
+              "data-cy": "trading-name-error"
             }
           }
-        ]
-      }) }}
-    </div>
+        }) }}
```

Comment:
Might be able to use `yesNoRadioButtons` component here to save repitition e.g https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/templates/insurance/eligibility/insured-amount.njk#L36

Reply:
done